external/vpx-centaur/README.md

```diff
@@ -0,0 +1,39 @@
+# Centaur (Bally 1981)
+
+![Table Preview](https://github.com/Bla1ze/vpx-images/blob/main/vpx-centaur.png)
+
+Author: [Bord](https://vpuniverse.com/profile/9265-bord/)  
+
+Version: 1.1
+
+Download: [VP Universe](https://vpuniverse.com/files/file/12948-centaur-bally-1981/)
+
+DirectB2S
+
+Author: [Manbearpig](https://vpuniverse.com/profile/32743-manbearpig/)  
+
+Version: 1
+
+Download: [VP Universe](https://vpuniverse.com/files/file/11861-centaur-bally-1981-b2s-full-dmd-and-3-screen/)
+
+ROM
+
+Download: [VP Forums](https://www.vpforums.org/index.php?app=downloads&showfile=658)
+
+SHA: 280162d48b728f2c3a06854b09f919e5efdee966
+
+MD5: a4dc0c753d1c1e5f6d573478a7d17cf2
+
+## Status 
+
+Minimum VPX Standalone build: 10.8.0-1983-a764013
+| Playfield | Controls | Backglass | DMD | FPS | 
+|-----------|----------|-----------|-----|----------|
+| :white_check_mark: | :white_check_mark: | :white_check_mark: | :x: | 40 |
```

Comment:
you're missing the rom section, and this table has a rom

Reply:
Updated.